PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+     For Work In Progress Pull Requests, please use the Draft PR feature,
+     see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
+
+     For a timely review/response, please avoid force-pushing additional
+     commits if your PR already received reviews or comments.
+
+     Before submitting a Pull Request, please ensure you've done the following:
+     - 👷‍♀️ Create small PRs. In most cases this will be possible.
+     - ✅ Provide tests for your changes.
+     - 📝 Use descriptive commit messages.
+     - 📗 Update any related documentation and include any relevant screenshots.
+-->
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation Update
+
+## Description
+
+
+## Related Tickets & Documents
+
+<!--
+For pull requests that relate or close an issue, please include them
+below.
+
+For example having the text: "closes READ-1234" would connect the current pull
+request to issue READ-1234 in your linked JIRA tenancy 
+-->
+
+- Related Issue #
+
+
+## Added/updated tests?
+
+- [ ] Yes
+- [ ] No, and this is why: _please replace this line with details on why tests
+      have not been included_
+- [ ] I need help with writing tests
+
+## [optional] Are there any post Release tasks we need to perform?
```

Comment:
A suggestion. Add a section to include an optional screenshot/screen recording of the change?

Eg:
```md
## [optional] Provide a screenshot or screen recording of your change
```

Reply:
The problem here is that once the PR is merged (e.g. to a develop branch, in preparation for a release, that sort of information will be lost. We definitely need to surface that sort of information somewhere, but in the PR doesn't feel like that place.